src/bindings/python/PyGradingData.cpp

```diff
@@ -0,0 +1,252 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#include <sstream>
+#include "PyOpenColorIO.h"
+#include "PyUtils.h"
+
+namespace OCIO_NAMESPACE
+{
+
+namespace
+{
+enum GradingDataIterator
+{
+    IT_CONTROL_POINT = 0,
+};
+
+using GradingControlPointIterator = PyIterator<GradingBSplineCurveRcPtr, IT_CONTROL_POINT>;
+
+void CopyGradingBSpline(GradingBSplineCurveRcPtr to, const ConstGradingBSplineCurveRcPtr from)
+{
+    const size_t numPt = from->getNumControlPoints();
+    to->setNumControlPoints(numPt);
+    for (size_t pt = 0; pt < numPt; ++pt)
+    {
+        to->getControlPoint(pt) = from->getControlPoint(pt);
+    }
+}
+}
+
+void bindPyGradingData(py::module & m)
+{
+    py::class_<GradingRGBM>(m, "GradingRGBM")
```

Comment:
TODO: Add constructor with keyword arg support to all Grading classes (can be done in later PR).

Reply:
Ok